### ISSUE TITLE
Accessibilty: Oprhaned or missing 'form labels' 

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#12]
WCAG 2.0- 1.1.1 Non-text Content Level (A) - violation 'All form inputs have associated text labels.

1.Go to: http://data.ottawa.ca/dataset/elected-officials
2.Observe the below image of identified wave tool 'red' image identifying either;
A. Missing form label missing - does not have a corresponding form label.
B. Orphaned form label - a form label is present but not associated with the form.

Resolution: Add text labels to all form inputs. Ensure all orphaned form labels are associated with the form.
![elected officials_orphaned or missing form labels](https://cloud.githubusercontent.com/assets/5271253/8041926/ab3120e8-0de9-11e5-8238-bfd276d1e0a2.JPG)
